source/features/clean-conversation-headers.tsx

```diff
@@ -27,14 +27,15 @@ async function initIssue(): Promise<void> {
 }
 
 async function initPR(): Promise<void> {
-	const byline = await elementReady('.gh-header-meta .flex-auto:not(.rgh-clean-conversation-header)');
-	if (!byline) {
+	const author = await elementReady('.gh-header-meta .flex-auto:not(.rgh-clean-conversation-header) .author');
+	if (!author) {
 		return;
 	}
 
+	const byline = author.closest<HTMLElement>('.flex-auto')!;
```

Comment:
Didn't we fix this at some point? Or did it never work?
```suggestion
	const byline = author.closest('.flex-auto')!;
```

Anyway the code LGTM

Reply:
The default is `Element` so unless I do `div.flex-auto`... I dont want to do that.

Reply:
https://github.com/sindresorhus/refined-github/issues/4008#issuecomment-823814526

Reply:
https://github.com/g-plane/typed-query-selector/issues/16